sdk/build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     annotationProcessor libraries.auto_value
 
     testAnnotationProcessor libraries.auto_value
+    testCompileOnly libraries.auto_value_annotation
```

Comment:
any idea why this was needed to be added? seems unrelated to the PR, on its face.

Reply:
I think Truth was leaking this in their POM so the test dependency on Truth got this transitively.